### ISSUE TITLE
Typescript support

### ISSUE THREAD

**Issue by gekkedev** [nuxt-community/fontawesome-module#27]
AFAIK there are no type definitions; or are they in a different module?

**Comment by pimlie** [comment]
@gekkedev No, I guess those are just missing. A pr to add them would be awesome!

**Comment by gekkedev** [comment]
PR created; pls take a look if you can.
